### PR TITLE
Bump to 1.11

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

This needs to wait until we have the 

```
registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.10.0:serverless-bundle
```

Image from Warren/CI

### CODE REVIEW

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml
Comment (markusthoemmes) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:37]:
```suggestion
    olm.skipRange: ">=1.10.0 <1.11.0"
```

Comment (matzew) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:37]: :facepalm:  good morning, Matthias :-) 

thanks

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve
/hold

Let's wait for all tests to pass before merging

File: olm-catalog/serverless-operator/Dockerfile
Comment (mgencur) [inline olm-catalog/serverless-operator/Dockerfile:13]: I'm curious if this file should target only OCP 4.6. The component matrix says 4.6 will be the only supported: https://docs.google.com/spreadsheets/d/1HTxR37_MM03_JQImW-KmAFk1oEQ83fXzfGxdfWQSX6o/edit#gid=0
However, current CI on 4.6 keeps failing due to issues with the index image.

Comment (markusthoemmes) [reply olm-catalog/serverless-operator/Dockerfile:13]: Maybe yes. Let's revisit after getting the initial bits in though to unblock some stuff that depends on the 0.17 bump.

**Comment by markusthoemmes** [comment]
/unhold

I mostly wanted to see the upgrade test passing. There's no reason the upstream tests would have issues.

**Pull request merged**
